pkg/skaffold/schema/defaults/defaults.go

```diff
@@ -96,7 +96,10 @@ func Set(c *latest.SkaffoldConfig) error {
 		return err
 	}
 
-	for _, pf := range c.PortForward {
+	for i, pf := range c.PortForward {
+		if pf == nil {
+			return fmt.Errorf("config's portForward[%d] is empty, Please check if it has valid values", i)
```

Comment:
Since it's possible for multiple configs to be tied together now/have multiple configs in the same file, we might want to try grabbing `c.Metadata.Name` so we can give them a little more information to the user with the error message

Reply:
Applied, Seems quite handy.